06-aks-cluster.md

```diff
@@ -120,7 +108,42 @@ Now that the [cluster prequisites and shared Azure service instances are provisi
 
         > :book: GitOps allows a team to author Kubernetes manifest files, persist them in their git repo, and have them automatically apply to their cluster as changes occur. This reference implementation is focused on the baseline cluster, so Flux is managing cluster-level concerns. This is distinct from workload-level concerns, which would be possible as well to manage via Flux, and would typically be done by additional Flux operators in the cluster. The namespace cluster-baseline-settings will be used to provide a logical division of the cluster bootstrap configuration from workload configuration. Examples of manifests that are applied: Cluster Role Bindings for the AKS-managed Azure AD integration, AAD Pod Identity, CSI driver and Azure KeyVault CSI Provider
 
-        > :warning: You can continue only after the GitHub Workflow completes successfully
+    1.  You can continue only after the GitHub Workflow completes successfully
+
+        ```bash
+        # Region 1 - Wait until updated with provisioningState at 'Succeeded'.
+        az deployment group wait -n cluster-stamp -g rg-bu0001a0042-03 --updated
+        # Region 2- Wait until updated with provisioningState at 'Succeeded'.
```

Comment:
have we considered `gh api /repos/:owner/:repo/actions/workflows` instead? 👀

Reply:
Response:
```
{
  "total_count": 1,
  "workflows": [
    {
      "id": 6532436,
      "node_id": "MDg6V29ya2Zsb3c2NTMyNDM2",
      "name": "Deploy AKS Secure Baseline clusters stamp and Flux in different regions",
      "path": ".github/workflows/aks-deploy.yaml",
      "state": "active",
      "created_at": "2021-03-09T09:00:30.000-03:00",
      "updated_at": "2021-03-09T09:00:30.000-03:00",
      "url": "https://api.github.com/repos/v-fearam/aks-secure-baseline/actions/workflows/6532436",
      "html_url": "https://github.com/v-fearam/aks-secure-baseline/blob/main/.github/workflows/aks-deploy.yaml",
      "badge_url": "https://github.com/v-fearam/aks-secure-baseline/workflows/Deploy%20AKS%20Secure%20Baseline%20clusters%20stamp%20and%20Flux%20in%20different%20regions/badge.svg"
    }
  ]
}
```
This is information about workflows defined, not executions states

Reply:
Thank for the support Fernando 
until export GH_WF_STATUS=$(gh api /repos/:owner/:repo/actions/runs/$(gh api /repos/:owner/:repo/actions/runs -q ".workflow_runs[0].id") -q ".status" 2> /dev/null) && [[ $GH_WF_STATUS == "completed" ]]; do echo "Monitoring GitHub workflow execution: ${GH_WF_STATUS}" && sleep 20; done